src/config/config.ts

```diff
@@ -108,10 +108,17 @@ export interface CustomizationConfig {
     languageData: any[];
 }
 
+export type FlashMessageType = 'error' | 'info' | 'warning' | 'success';
+
+export interface FlashMessage {
+    type: FlashMessageType;
+    message: string;
```

Comment:
modifying API to be strings as per https://github.com/bigcommerce/bigcommerce/pull/34969